src/components/File.vue

```diff
@@ -335,14 +334,14 @@ export default {
 		width: fit-content;
 
 		// Make the checkbox background round on hover.
```

Comment:
Right here, a nitpick is that now we want the background to always be present for accessibility reasons, no? So the comment could remove the hover part